streampipes-model/src/main/java/org/apache/streampipes/model/configuration/DefaultEmailTemplateConfiguration.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.streampipes.model.configuration;
+
+public class DefaultEmailTemplateConfiguration {
+
+  public EmailTemplateConfig getDefaultTemplates() {
+    return new EmailTemplateConfig(getTemplate());
+  }
+
+  @SuppressWarnings("all")
+  private String getTemplate() {
+    return "<!DOCTYPE html>\n"
```

Comment:
How about moving this HTML code to a dedicated file? This would be in line with `streampipes-mail`

Reply:
done